composables/useProfile.ts

```diff
@@ -2,7 +2,7 @@ import { useQueries } from '@tanstack/vue-query'
 
 import { browserProcessMetadata } from '@/utils/processMetadata'
 
-import type { LSP3ProfileMetadataJSON } from '@lukso/lsp-smart-contracts'
+import type { LSP3ProfileMetadataJSON } from '@lukso/lsp3-contracts'
```

Comment:
You can revert back to importing from `@lukso/lsp-smart-contracts`.
In the rc.5, I included so that it is backward compatible with the previous import syntax :)